docs/excel/custom-functions-requirement-sets.md

```diff
@@ -12,11 +12,17 @@ localization_priority: Normal
 
 |  Requirement set  |  Office on Windows<br>(connected to a Microsoft 365 subscription)  |  Office on iPad<br>(connected to a Microsoft 365 subscription)  |  Office on Mac<br>(connected to a Microsoft 365 subscription)  | Office on the web |
 |:-----|-----|:-----|:-----|:-----|:-----|
-| CustomFunctionsRuntime 1.1 | 16.0.12527.20092 or later | Not supported | 16.34 or later | May |
+| CustomFunctionsRuntime 1.1 | 16.0.12527.20092 or later | Not supported | 16.34 or later | May 2019 |
+| CustomFunctionsRuntime 1.2 | 16.0.12527.20194 or later | Not supported | 16.34.20020900 or later | January 2019 |
+| CustomFunctionsRuntime 1.3 | 16.0.13127.20296 or later | Not supported | 16.40.20081000 or later | July 2020 |
 
 > [!NOTE]
 > Excel custom functions are not supported on Office 2019 or earlier (one-time purchase).
 
+## CustomFunctionsRuntime 1.1, 1.2, and 1.3
+
+The CustomFunctionsRuntime 1.1 is the first version of the API. Version 1.2 adds the `CustomFunctions.Error` object to support error handling. Version 1.3 adds [XLL streaming](/make-custom-functions-compatible-with-xll-udf#custom-function-behavior-for-xll-compatible-functions) support and new `ErrorCode` options to the `CustomFunctions.Error` object. 
```

Comment:
Instead of referring to these as "versions", use the term "requirement set". Version implies the old stuff is getting replaced, whereas here it's getting added to.

Reply:
I updated two of the references, but I left "CustomFunctionsRuntime 1.1 is the first version" because it seems more accurate based on your description. Could you take a look and let me know what you think?